### ISSUE TITLE
Visual bug in local library progress bar

### ISSUE THREAD

**Issue by rkben** [nukeop/nuclear#1644]
**Platform:**
macOS Sonoma 14.5, arm64

**Nuclear version:**
git master: https://github.com/nukeop/nuclear/commit/022ba2b98302939222bbe7dabb397ef0934c3b51

**Description of the issue:**
Visual bug in local library progress bar. I suspect this is due to the width of the characters in the total number `34857`. The currently scanned count is not viewable at all until its width is 4 characters wide.

![image](https://github.com/user-attachments/assets/f72d7c8f-16bb-4484-b168-7c287b2e3824)

**Comment by nukeop** [comment]
You're right, we're using this library: https://react.semantic-ui.com/modules/progress/

I guess this can be handled by CSS. By the way, did you notice any improvement in the scanning time? Did it handle your library well?

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
